### ISSUE TITLE
Pan Dismiss Direction

### ISSUE THREAD

**Issue by anthonyuclvcfg** [Tkko/Flutter_dismissible_page#21]
pan - can be dismissed by dragging either horizontal or vertical, no multi

great job !

**Comment by Tkko** [comment]
I don't get what do you mean 😕

**Comment by anthonyuclvcfg** [comment]
yes sorry friend, I was talking you can include a new dismiss direction as an enhancement -> Pan 
this is a bit more restrictive than multi, you can only do a horizontal or vertical movement

https://docs.flutter.dev/development/ui/advanced/gestures

**Comment by Tkko** [comment]
There is an options for that `DismissiblePageDismissDirection.vertical` and `DismissiblePageDismissDirection.horizontal`

**Comment by dionysos1** [comment]
What I believe he means and what I would like to see as a feature, Is to be able to use both the vertical and horizontal dismiss direction but not have the multi functionality of dragging it around like a card.
So essentially locking in in the direction you are swiping. And also the ability to disable the gesture on 1 of the directions (for me this is a swipe from the bottom that I would like to disable)

**Comment by Tkko** [comment]
@dionysos1 Disabling specific direction is on top of my tasks already, but my time is pretty limited these days so if anyone is open to contribute I'll be happy to help.
